docs/cugraph/source/graph_support/algorithms/jaccard_similarity.md

```diff
@@ -0,0 +1,26 @@
+# Jaccard Similarity
+
+The Jaccard similarity between two sets is defined as the ratio of the volume of their intersection divided by the volume of their union. 
+
+The Jaccard Similarity can then be defined as
+
+Jaccard similarity coefficient = |A ∩ B| / |A ∪ B|
```

Comment:
use the math tool to make equations better

Reply:
done